src/components/AssetsSearch/AssetsSearch.scss

```diff
@@ -0,0 +1,9 @@
+.form-inline label {
+  margin-right: 10px;
```

Comment:
I think this could be `$spacer` just in case someone wants to theme spacing, sort of like what we do in https://github.com/edx/studio-frontend/blob/master/src/components/AccessibilityPolicyForm/AccessibilityPolicyForm.scss#L4

Reply:
So `$spacer` ends up being 16px because of the weird `base-rem-size` Studio has. I could do `1rem` instead for 10px and `0.8rem` for 8px. It would still be kinda configurable if they changed the `base-rem-size` (everything would scale with that).